games-strategy/wesnoth/wesnoth-1.14.1.ebuild

```diff
@@ -12,14 +12,15 @@ SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86 ~x86-fbsd"
-IUSE="dbus dedicated doc fribidi nls openmp server"
+IUSE="dbus dedicated doc fribidi libressl nls openmp server"
 
 RDEPEND="
 	>=dev-libs/boost-1.50:=[nls,threads,icu]
 	>=media-libs/libsdl2-2.0.4:0[joystick,video,X]
 	!dedicated? (
 		dev-libs/glib:2
-		dev-libs/openssl:0=
+		!libressl? ( dev-libs/openssl:0 )
+		libressl? ( dev-libs/libressl )
```

Comment:
This requires the slot and subslot modifiers to be present.

e.g. dev-libs/libressl:0=

This ensures rebuilds are properly triggered on library upgrades/downgrades.  Same for dev-libs/openssl.

Reply:
Thanks Aaron.
I've fixed this and updated the patch.